library/spdm_requester_lib/libspdm_req_negotiate_algorithms.c

```diff
@@ -195,6 +195,13 @@ static libspdm_return_t libspdm_try_negotiate_algorithms(libspdm_context_t *spdm
         status = LIBSPDM_STATUS_INVALID_MSG_FIELD;
         goto receive_done;
     }
+    if (spdm_request->header.spdm_version >= SPDM_MESSAGE_VERSION_12) {
+        if (!libspdm_onehot0(spdm_response->other_params_selection &
+                             SPDM_ALGORITHMS_OPAQUE_DATA_FORMAT_MASK)) {
```

Comment:
This check is not enough, because we dont want to support `SPDM_ALGORITHMS_OPAQUE_DATA_FORMAT_0`.

I think it should be "((selection & MASK) != FORMAT_NONE) && ((selection & MASK) != FORMAT_1)"

Reply:
`SPDM_ALGORITHMS_OPAQUE_DATA_FORMAT_0` is fine for libspdm. It just means that the Integrator needs to provide / parse all of the opaque data and they can't do key exchange.